app/src/main/java/com/example/poste/activities/DashboardActivity.java

```diff
@@ -287,10 +287,17 @@ public void onClick(DialogInterface dialog, int which) {
                         public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response) {
                             if (response.isSuccessful()) {
                                 // create a local folder matching the folder created by the api.
-                                currentUser.updateFoldersAndPosts(DashboardActivity.this);
-                                userFolders = currentUser.getFolders();
-                                folderAdapter.notifyItemInserted(userFolders.size() - 1);
+                                User.getUser().updateFoldersAndPosts(DashboardActivity.this);
+                                // userFolders = currentUser.getFolders();
+                                // folderAdapter.notifyItemInserted(userFolders.size() - 1);
```

Comment:
Looks good to me.
All I can suggest is delete these commented out lines. And pull in Dev make sure this branch is a fast forward.